README.md

```diff
@@ -14,7 +14,28 @@ https://www.gov.uk/api/search.json?q=taxes
 
 ## Technical documentation
 
-This is a Sinatra application that provides an API for search to multiple applications. It isn't supposed to be used by regular users, but it is publicly available on [gov.uk/api/search.json](https://www.gov.uk/api/search.json?q=taxes).
+Rummager is a Sinatra application that interfaces with Elasticsearch.
+
+It provides a [search API](docs/unified-search-api.md) that is used by multiple applications, and is publicly available at [gov.uk/api/search.json](https://www.gov.uk/api/search.json?q=taxes).
+
+There are two ways documents get added to the search index:
+
+1. Post to the [Documents API](docs/documents.md)
+2. Via the RabbitMQ consumer worker, which responds to notifications from the [Publishing API](https://github.com/alphagov/publishing-api).
```

Comment:
Perhaps leave this out for now, because it isn't actually true atm. We'll add it back in when we activate the queue.